docusaurus/docs-internals/editor-controls/data-controls/inputText.md

```diff
@@ -0,0 +1,44 @@
+| :--------------: | :-----: | :-----: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       `id`       | string  |    -    | The identifier of the key where the inputText will save your data.                                                                                                                                                                                                               |
+|      `type`      | string  |    -    | Type should be `"inputText"` to use this control.                                                                                                                                                                                                                                |
+|  `placeholder`   | string  |    -    | The placeholder text displayed in the input field.                                                                                                                                                                                                                               |
+|     `label`      | string  |    -    | The label displayed on the left side of the control.                                                                                                                                                                                                                             |
+|    `position`    | number  |    -    | The position of the control in the toolbar.                                                                                                                                                                                                                                      |
+|    `devices`     | string  | `"all"` | Define the devices where the control will be rendered. Options: `"all"`, `"desktop"`, `"responsive"`. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+|    `disabled`    | boolean |  false  | Determines whether the control is disabled or enabled.                                                                                                                                                                                                                           |
+| `helper.content` | string  |    -    | A description placed on the help icon to assist the user.If there is help content, the icon is displayed                                                                                                                                                                         |
```

Comment:
1. helper also has the position

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/2208e303-66dd-4423-bf47-5fa06e58f94f)

2. add `helper.content` and `helper.position` to the Switch also